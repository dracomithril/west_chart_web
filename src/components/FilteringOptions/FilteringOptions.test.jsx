import React from 'react';
import Enzyme, { shallow } from 'enzyme';
import Adapter from 'enzyme-adapter-react-16';
import { shallowToJson } from 'enzyme-to-json';
import { componentWithStyles as FilteringOptions } from './index';

Enzyme.configure({ adapter: new Adapter() });

describe('<FilteringOptions/>', () => {
  it('renders without crashing FilteringOptions', () => {
    const wrapper = shallow(
      <FilteringOptions />,
    );
    expect(shallowToJson(wrapper)).toMatchSnapshot();
  });
});
