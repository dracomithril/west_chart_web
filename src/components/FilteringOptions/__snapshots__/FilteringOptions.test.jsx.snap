// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<FilteringOptions/> renders without crashing FilteringOptions 1`] = `
<FilteringOptions
  classes={
    Object {
      "button": "FilteringOptions-button-1",
      "details": "FilteringOptions-details-4",
      "heading": "FilteringOptions-heading-2",
      "root": "FilteringOptions-root-3",
    }
  }
/>
`;
